docs/Developers/Language_Naming_Format.rst

```diff
@@ -0,0 +1,9 @@
+.. note::
+
+    - For language naming, always consider the format that is used in the
+      corresponding Wikipedia page (exception: use CPP instead
+      of cpp or c++);
```

Comment:
Is the use of `;` fine here?

Reply:
@frextrite Actually the points were provided by @ayan-b in the issue description.I also thought about the semi colon and I think it should'nt be there .What do you guys think ?

Reply:
If you think so, you can remove. Its your PR after all. :wink: 
I gave the points for purpose of references only, if you wish, you can change them or elaborate them. :smile:

Reply:
@ayan-b So sorry I just misunderstood that :sweat_smile: